maintainers/metrics.nix

```diff
@@ -0,0 +1,84 @@
+{
+  lib,
+  ngipkgs,
+  raw-projects,
+}:
+let
+  inherit (lib)
+    attrNames
+    concatMap
+    attrValues
+    filterAttrs
+    mapAttrs
+    count
+    optionalAttrs
+    ;
+in
+rec {
+  metrics = {
+    projects = attrNames raw-projects;
+    in-ngipkgs = attrNames ngipkgs;
+    derivations = concatMap (p: attrNames p.packages) (attrValues raw-projects);
```

Comment:
Should we remove `derivations`, since we're no longer tracking `packages`?

Reply:
Let's keep it, it's the only metric that stayed consistent since the beginning, and while it's a weak proxy it still gives us an indication of how complex a project is.